devkit/README.md

```diff
@@ -0,0 +1,96 @@
+
+|  Project                                         | Startup option                                                                                                  |
+|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| [`csi-baremetal-devkit`](https://github.com/dell/csi-baremetal)                                 | `./devkit` (if installed: `devkit`)                    |
+
+### Configuring IDEs  
+Currently devkit supports IntelliJ IDEA and CLion. These IDEs can be started inside devkit. 
+- You should set variables as described in `bashrc_templates` for your case.  
+- Add `--idea yes` and/or `--clion yes` to startup options for start your IDE. You can also start both when you are inside the devkit by typing `idea/clion` correspondingly.
+- By default, it is assumed that the sources of IDE installed to `/opt/<ide_name>`. But it can be configured with setting `DEVKIT_SOFTWARE_HOST_DIR_PATH` to your location. 
+- It is also possible that you need to add permissions to this directory to avoid errors:`sudo chmod -R a+rw /opt/<ide_name>`.  
+
+### Files Description
+|  File                                            | Description                                                                                                                          |
+|--------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
+| [`Dockerfile`](./Dockerfile)                     | One of the input arguments to `docker build` command (see [Dockerfile reference](https://docs.docker.com/engine/reference/builder)). |
+| [`devkit`](./devkit)                             | Wrapper around `docker run` command that hides complex list of arguments from the developer.                                         |
+| [`devkit-entrypoint.sh`](./devkit-entrypoint.sh) | Docker image entrypoint. Performs required preparations and starts required services.                                                |
+| [`start_ide`](./start_ide)                       | Script to start IDE in a background.                                                                                       |
+
+### Versioning
+Devkit follows the semantic versioning approach ([semver.org](http://semver.org/)).
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+> 1. MAJOR version when you make incompatible API changes,
+> 2. MINOR version when you add functionality in a backwards-compatible manner, and
+> 3. PATCH version when you make backwards-compatible bug fixes.
+
+
+### Examples
+*Note* that the following examples may be a bit outdated in terms of used environment variables and input options.
+
+#### Developer Opens an Interactive Bash Session
+```
+user@emc:~> devkit
+DEVKIT: viprhal and viprhal-devel....[+]
```

Comment:
Looks like you have wrong output here. There is no such package called `viprhal` and `viprhal-devel`